client/app/scripts/charts/node-networks-overlay.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import d3 from 'd3';
+import { List as makeList } from 'immutable';
+import { getNodeColor } from '../utils/color-utils';
+import { isContrastMode } from '../utils/contrast-utils';
+
+
+const h = 5;
+const padding = 0.05;
+const rx = 1;
+const ry = rx;
+const labelOffset = 38;
+
+function NodeNetworksOverlay({size, stack, networks = makeList()}) {
+  const r = size * 0.5;
+  const offset = r + labelOffset;
+  const w = Math.max(size, (size / 4) * networks.size);
+  const x = d3.scale.ordinal()
```

Comment:
It could! It feels bad modifying a global scale like that. Much prefer this style but looking through the scale src it is doing a little bit more work.